mne/preprocessing/ica.py

```diff
@@ -224,7 +224,17 @@ class ICA(ContainsMixin):
     def __init__(self, n_components=None, max_pca_components=None,
                  n_pca_components=None, noise_cov=None, random_state=None,
                  method='fastica', fit_params=None, max_iter=200,
-                 verbose=None):  # noqa: D102
+                 verbose=None, channel_start=None):  # noqa: D102
+
+        if channel_start is None:
+            channel_start = 1
+            warn('channel_start will default to 1 in 0.15, but change '
+                 'to 0 in 0.16. Set it explicitly to avoid this warning',
+                 DeprecationWarning)
+        if channel_start not in (0, 1):
+            raise ValueError('channel_start must be 0 or 1, got %s' %
+                             (channel_start,))
```

Comment:
I'd opt for not deprecating. We could even twitter vote ... Let's just get it right once.